src/pages/ExpressionMaker/ExpressionMaker.tsx

```diff
@@ -0,0 +1,88 @@
+import { RouteProp } from '@react-navigation/native'
+import React, { useState } from 'react'
+import { StyleSheet, View } from 'react-native'
+import { Button, List } from 'react-native-paper'
+import { RootStackParamList } from '../../App'
+import { ExpressionDisplay } from '../../components/expressions/ExpressionDisplay'
+import {
+	NumberInputModal,
+	TextInputModal,
+} from '../../components/expressions/LiteralModal/LiteralInputModals'
+import moveToBottom from '../../components/ui/MoveToBottom'
+import { useExpression } from '../../context/ExpressionProvider'
+import { Literal } from '../../models/expression/segments'
+import { translate } from '../../utils/translation-helpers'
+
+export type ExpressionMakerProp = RouteProp<
+	RootStackParamList,
+	'ExpressionMaker'
+>
+
+function ExpressionMaker() {
+	// const { project } = useProject()
+	const {
+		expression,
+		actions: { reset, addSegment },
+	} = useExpression()
+	const [showNumberModal, setShowNumberModal] = useState(false)
+	const [showTextModal, setShowTextModal] = useState(false)
+
+	return (
+		<View style={container}>
+			{expression.segments.length > 0 ? (
+				<List.Section>
+					<List.Subheader>Mensajes</List.Subheader>
+					{/* {last(expression.segments)!.methods.map(m => (
+						<View key={m.name}>
+							<List.Item
+								title={m.name}
+								onPress={() => addSegment(new MethodSegment(m))}
+							/>
+						</View>
+					))} */}
+				</List.Section>
+			) : (
+				<List.Section>
+					<List.Subheader>Objetos</List.Subheader>
+					{/* {project.entities.map(e => (
+						<View key={e.name}>
+						<List.Item title={e.name} onPress={() => addSegment(e)} />
+						</View>
+					))} */}
+					<List.Subheader>Literales</List.Subheader>
+					<List.Item
+						title={translate('number')}
+						onPress={() => setShowNumberModal(true)}
+					/>
+					<List.Item
+						title={translate('string')}
+						onPress={() => setShowTextModal(true)}
+					/>
+					<List.Item
+						title="True"
+						onPress={() => addSegment(new Literal(true))}
+					/>
+					<List.Item
+						title="False"
+						onPress={() => addSegment(new Literal(false))}
+					/>
+				</List.Section>
+			)}
+			<Button onPress={reset}>RESET</Button>
+			{moveToBottom(
+				<ExpressionDisplay displayColor="white" expression={expression} />,
+			)}
```

Comment:
`moveToBottom` significa "mostralo abajo", no? No se podría escribir onda:
```suggestion
			<BottonView>
				<ExpressionDisplay displayColor="white" expression={expression} />,
			</BottonView>
```

Igual lo pondría arriba del listado, no abajo, se pierde (podemos hacer una prueba con gente para ver qué opina).

Reply:
lo movi parriba